<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rescoreMmpbsaDialog</class>
 <widget class="QDialog" name="rescoreMmpbsaDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MM/PBSA Rescoring Parameters</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>10</y>
     <width>191</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Rescoring with MM/PBSA</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_amber">
   <property name="geometry">
    <rect>
     <x>486</x>
     <y>40</y>
     <width>86</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>240</y>
     <width>561</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ok">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>260</y>
     <width>86</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_model">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>217</y>
     <width>86</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;- plp95: Piecewise Linear Potential from Gehlhaar DK et al&lt;/p&gt;&lt;p&gt;- plp: PLANTS version of the Piecewise Linear Potential&lt;/p&gt;&lt;p&gt;- chemplp: PLANTS version of the Piecewise Linear Potential implementing GOLD's terms&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <item>
    <property name="text">
     <string>PB3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>GB5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>GB8</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>218</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Model</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>260</y>
     <width>86</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_amber">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>40</y>
     <width>351</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_exec">
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>40</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>amber.sh File</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>30</y>
     <width>20</width>
     <height>221</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>203</x>
     <y>10</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_MD">
   <property name="geometry">
    <rect>
     <x>486</x>
     <y>216</y>
     <width>86</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Configure</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_MD">
   <property name="geometry">
    <rect>
     <x>325</x>
     <y>217</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Run a short implicit solvent (GB) MD&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Short MD simulation</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>541</width>
     <height>137</height>
    </rect>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContents</enum>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>100</number>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>1</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <row>
    <property name="text">
     <string>PB3</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>GB5</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>GB8</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>Model</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Radii for Gpol</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>SASA for Gnp</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Surface Tension (γ)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Surface Offset (β)</string>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string>PB3</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="0" column="1">
    <property name="text">
     <string>Parse</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="0" column="2">
    <property name="text">
     <string>Molsurf</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="0" column="3">
    <property name="text">
     <string>0.00542</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="0" column="4">
    <property name="text">
     <string>0.92</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="1" column="0">
    <property name="text">
     <string>GB5</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="1" column="1">
    <property name="text">
     <string>mbondi2</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="1" column="2">
    <property name="text">
     <string>LCPO</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="1" column="3">
    <property name="text">
     <string>0.00500</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="1" column="4">
    <property name="text">
     <string>0</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="2" column="0">
    <property name="text">
     <string>GB8</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="2" column="1">
    <property name="text">
     <string>mbondi3</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="2" column="2">
    <property name="text">
     <string>LCPO</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="2" column="3">
    <property name="text">
     <string>Atom dependant</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="2" column="4">
    <property name="text">
     <string>0.195141</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
